lib/load.php

```diff
@@ -188,28 +122,6 @@ function gutenberg_is_experiment_enabled( $name ) {
 // Fonts API / Font Face.
 remove_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' ); // Turns off WordPress 6.0's stopgap handler.
```

Comment:
@hellofromtonya, do we still need this?

Reply:
@Mamaduka no longer needed as of 6.4.0. 

[That callback was deprecated ](https://core.trac.wordpress.org/changeset/56500#file4)and its associated `add_action()` [hook](https://core.trac.wordpress.org/changeset/56500#file3) was removed in 6.4.0 via [r56500](https://core.trac.wordpress.org/changeset/56500).

Reply:
Thank you, @hellofromtonya!